a/pkg/virt-operator/resource/generate/components/daemonsets.go b/pkg/virt-operator/resource/generate/components/daemonsets.go
index da66b6cd9..aebc64605 100644
--- a/pkg/virt-operator/resource/generate/components/daemonsets.go
+++ b/pkg/virt-operator/resource/generate/components/daemonsets.go
@@ -320,8 +320,10 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 
 	container.Resources = corev1.ResourceRequirements{
 		Requests: corev1.ResourceList{
-			corev1.ResourceCPU:    resource.MustParse("10m"),
-			corev1.ResourceMemory: resource.MustParse("325Mi"),
+			corev1.ResourceCPU: resource.MustParse("10m"),
+			// 325Mi - base memory request
+			// +32Mi - to account for the buffer used to verify containerdisk checksums
+			corev1.ResourceMemory: resource.MustParse("357Mi"),
 		},
 	}
 	if prHelperImage == "" {
